neo/core/epoch.py

```diff
@@ -19,13 +19,15 @@
 PY_VER = sys.version_info[0]
 
 def _new_epoch(cls, times=None, durations=None, labels=None, units=None,
-                name=None, description=None, file_origin=None, annotations = None):
+                name=None, description=None, file_origin=None, annotations = None, segment=None):
     '''
-    A function to map Event.__new__ to function that
+    A function to map epoch.__new__ to function that
     does not do the unit checking. This is needed for pickle to work. 
     '''
-    return Epoch( times=times, durations=durations, labels=labels, units=units, name=name, file_origin=file_origin,
+    e=Epoch( times=times, durations=durations, labels=labels, units=units, name=name, file_origin=file_origin,
```

Comment:
Please use spaces around "=" (except for function keyword arguments) (https://www.python.org/dev/peps/pep-0008/#other-recommendations)